exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = roomMaxCapacity >= moreStudentsThanMentors; // finish this statement
```

Comment:
1. The code runs but the logic needs refinement. i.e. we should ask whether the room have enough capacity to host all mentors and students.
2. The >= comparison "works" because the boolean variable moreStudentsThanMentors is converted to 1 for comparison. So really needs to watch out when coding in JS